### ISSUE TITLE
Audio recorder should be extended with file converter functionality

### ISSUE THREAD

**Issue by andorfermichael** [crowdbotics-users/audvice-native-audio-recorder#26]
**Is your feature request related to a problem? Please describe.**
Currently the audio recorder produces an audio file of wav format which is not that handy for further usage. 

**Describe the solution you'd like**
The audio recorder should get a new method for converting/processing the recorded wav files. 

The requirements are the following:

- Encoder: ffmpeg
- Format: AAC-LC
- Container: m4a
- Bitrate: 96kB

The method definition should be as follows:

`compress(file) -> Promise("file.m4a")`

The usage of the method should work like that:

`this.audioRecorderRef.compress(trackA).then(newCompressedTrack => console.log(newCompressedTrack))`

```
Promise.all(
    this.audioRecorderRef.compress(trackA),
    this.audioRecorderRef.compress(trackB),
    this.audioRecorderRef.compress(trackC),
).then(([newTrackA, newTrackB, newTrackC]) => console.log(newTrackA, newTrackB, newTrackC))
```

**Comment by ateqpm** [comment]
@andorfermichael just confirming
Does this issue includes? ,
1) app records even if phones goes to sleep/background view?
2) App records even when you navigate away from the audvice app? thanks

**Comment by andorfermichael** [comment]
@scbopha @ateqpm iOS is working and thus resolved by #30

**Comment by andorfermichael** [comment]
Resolved by #30 and #31
